### PR TITLE
installs vault e2e via release package

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -53,8 +53,8 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/certs"
 
-[[constraint]]
-  branch = "master"
+[[override]]
+  branch = "namespace"
```
Comment (kopiczko) [inline Gopkg.toml]: I think it's merged now.

File: integration/setup/config.go [hunk 2]
```diff
@@ -81,6 +84,19 @@ func NewConfig() (Config, error) {
 		}
 	}
 
+	var k8sSetup *k8s.Setup
```
Comment (kopiczko) [inline integration/setup/config.go]: I understand why this name is better now. `k8sClient` would collide a lot.

**Comment by kopiczko** [comment]
I will update e2esetup here.

**Comment by xh3b4sd** [comment]
E2E seem fine. Always other tests fail with the API not being available. Known issue. Rolling.

**Pull request merged**